gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -4275,7 +4309,7 @@ subroutine GFS_physics_driver                                 &
                           Statein%pgr, Statein%phil, clw, Stateout%gq0,     &
                           Stateout%gt0, Stateout%gu0, Stateout%gv0, rain1,  &
                           kbot, ktop, kcnv, islmsk, Statein%vvl, ncld,      &
-                          Diag%hpbl, hflx, evap, ud_mf, dt_mf, cnvw, cnvc,  &
+                          Diag%hpbl, hflxq, evapq, ud_mf, dt_mf, cnvw, cnvc,&
```

Comment:
I am sure you have a reason for it, but I don't understand why this change is only for shalcnv, but not for samfshalcnv (or any other convection scheme)?

Reply:
It is because only shalcnv among convection schemes needs kinematic sensible heat flux (hflx) and latent heat flux (evap) for the cloud base mass flux. The other convection schemes do not use hflx & evap.

Reply:
Thanks, got it! We need to make sure - I'll create an issue to check - that none of the additional convection schemes in CCPP that are not in IPD use hflx & evap.

Reply:
See https://github.com/NCAR/ccpp-physics/issues/443